packages/babel-parser/test/fixtures/placeholders/variable/using-init/output.json

```diff
@@ -1,6 +1,9 @@
 {
   "type": "File",
   "start":0,"end":30,"loc":{"start":{"line":1,"column":0,"index":0},"end":{"line":3,"column":1,"index":30}},
+  "errors": [
+    "SyntaxError: Using declaration cannot have destructuring patterns. (2:8)"
+  ],
```

Comment:
This is a valid case, when the placeholder plugin is enabled, `using %%name%% = null` is a valid production. Can we throw the error when `id` is either `ArrayPattern` or `ObjectPattern`?

Reply:
Hello @JLHwung! Thank you for your feedback. As you said, modified it to throw an error only in the case of ObjectPattern and ArrayPattern.